src/utils/_utils/mapSearch.js

```diff
@@ -0,0 +1,27 @@
+export const eventFinder = (events, query) => {
+    const matchingEvents = [];
+    events.forEach((event) => {
+        if(event.name.toLowerCase().search(query.toLowerCase()) !== -1) {
+
+            // Transform into carmen GeoJSON format
+            // https://github.com/mapbox/carmen/blob/master/carmen-geojson.md
+            const matchingEvent = {
+                type: "Feature",
+                // id: '', (we might need this, might not)
+                properties: {
+                    title: event.venue.name,
+                    description: event.description,
+                },
+                geometry: event.venue.location,
+                text: event.venue.name,
+                place_name: event.name,
+                center: event.venue.location.coordinates,
+                place_type: ['event'], // figure out what to put here
```

Comment:
where are you getting the structure of this event because it's different from the structure of the event list we have

Reply:
Can we clearly document the structure we're supposed to follow or otherwise we should delay this until the api is completely integrated with this repo.

Reply:
This is the basis that I'm working on api-blueprints! but the schema hasn't ever been changed - documentation is currently in [notion](https://www.notion.so/Event-Schemas-bb0e5f634029446d9c6da18cce4d2022).

Sorry that what's in notion might be a bit outdated but you could also try opening our [mongo atlas](https://github.com/onlinkers/event-discovery-api#mongodb) or, since we are aspiring **software developers**, by reading code and looking at the schema itself in [api](https://github.com/onlinkers/event-discovery-api/blob/master/src/models/event.js). It's all there, just takes a few seconds of diggin!

Reply:
We can't test this right now since the api is down tho?

Reply:
In any case the structure seems alright, but I think we can change the `type: "Feature"` to anything we want.